### PR TITLE
sys-firmware/sigrok-firmware-fx2lafw: backport syntax updates

### PR DESCRIPTION
**Pull request by mjsir911**
I've also pinged them to maybe do a new release on the mailing list

### CODE REVIEW

File: sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch [hunk 1]
```diff
@@ -0,0 +1,977 @@
+commit 96b0b476522c3f93a47ff8f479ec08105ba6a2a5
```
Comment (ceamac) [inline sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch:1]: Please add at the top the original URI for each patch, for example `http://sigrok.org/gitweb/?p=sigrok-firmware-fx2lafw.git;a=commitdiff_plain;h=96b0b476522c3f93a47ff8f479ec08105ba6a2a5` or whatever you used to obtain it.

**Review by ceamac — CHANGES_REQUESTED** [review]
The patch is quite large (it passes our 20k limit), so I cannot merge it.  I hope it's ok with you if I copy it to my dev space and adjust the ebuild to download it from there.

Thank you!

File: sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch (hunk 1 above)
Comment (mjsir911) [reply sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch:1]: Appending them together is still fine yeah?

Comment (mjsir911) [reply sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch:1]: also if you could point me to any documentation around this convention or something if it exists I'de much appreciate it

**Comment by mjsir911** [comment]
> The patch is quite large (it passes our 20k limit), so I cannot merge it. I hope it's ok with you if I copy it to my dev space and adjust the ebuild to download it from there.
> 
> Thank you!

yeah this would be much appreciated

File: sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch (hunk 1 above)
Comment (ceamac) [reply sys-firmware/sigrok-firmware-fx2lafw/files/sigrok-firmware-fx2lafw-update-syntax.patch:4]: We have some guidelines in the manual, please read the section on [clean patches](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#clean-patch-howto).  We need to know where the patch comes from, and a link to upstream is best in this case.

**Review by ceamac — APPROVED** [review]

**Pull request closed without merging**